docs/resources/aws_guardduty_detectors.md

```diff
@@ -0,0 +1,64 @@
+---
+title: About the aws_guardduty_detectors Resource
+platform: aws
+---
+
+# Requirements
+
+Train AWS version 0.1.18 or newer is required for this resource.
+
+# aws\_guardduty\_detectors
+
+Use the `aws_guardduty_detectors` InSpec audit resource to test properties of some or all AWS GuardDuty Detectors.
+
+## Syntax
+
+ Ensure you have exactly 3 Detectors available to you.
+
+    describe aws_guardduty_detectors do
+      its('detector_ids.count') { should cmp 3 }
+    end
+    
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on GuardDuty Detectors](https://docs.aws.amazon.com/guardduty/latest/ug/what-is-guardduty.html).
+
+## Properties
+
+|Property                 | Description|
+| ---                     | --- |
+|detector_ids             | The architecture of the image. |
+
+## Examples
+
+##### Ensure ID of an AMI GuardDuty Detector
+    describe aws_guardduty_detectors do
+      its('detector_ids') { should include ['image-id-43542'] }
+    end
+
+## Matchers
+
+For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/). 
+
+#### exist
+
+The control will pass if the describe returns at least one result.
+
+Use `should_not` to test the entity should not exist.
+
+    describe aws_guardduty_detectors.where( <property>: <value>) do
```

Comment:
Does this work in this way? Considering `aws_guardduty_detectors` only returns the `detector-ids`?

Reply:
Perhaps an example similar to [this](https://github.com/inspec/inspec-aws/blob/6c95770a30f2f8c1551ee543045dd56e063b06b1/docs/resources/aws_ebs_volumes.md#use-the-inspec-resource-to-request-the-ids-of-all-ebs-volumes-then-test-in-depth-using-aws_ebs_volume-to-ensure-all-volumes-are-encrypted-and-have-a-sensible-size) is more what we want here?

Reply:
Updated in latest commit.

Reply:
It would be good to add the example of using the plural and singular resource into the doc. Could just copy what you have in the integration tests